client/blocks/reader-post-card/featured-asset.jsx

```diff
@@ -2,35 +2,55 @@
  * External Dependencies
  */
 import React from 'react';
-import { head, filter, get } from 'lodash';
+import { head, filter } from 'lodash';
 
 /**
  * Internal Dependencies
  */
 import FeaturedVideo from './featured-video';
 import FeaturedImage from './featured-image';
 
+// TODO delete canonical image logic from rules?
+const candidateForFeature = ( media ) => {
```

Comment:
posts are coming from both flux and redux. Making it a selector wouldn't cover the flux sourced posts.